compiler/vm.nim

```diff
@@ -1355,6 +1355,21 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
         stackTrace(c, tos, pc,
           errIllegalConvFromXtoY % [
              $regs[ra].regToNode, "[" & $regs[rb].regToNode & ".." & $regs[rc].regToNode & "]"])
+    of opcRangeChckU:
+      let rb = instr.regB
+      let rc = instr.regC
+      # cast to unsigned
+      var raNode = regs[ra].regToNode
+      raNode.kind = nkUIntLit
+      var rbNode = regs[rb].regToNode
+      rbNode.kind = nkUIntLit
+      var rcNode = regs[rc].regToNode
+      rcNode.kind = nkUIntLit
```

Comment:
This is a bit ugly and generates warnings, so let me know if you have a better idea.